src/components/Hero.tsx

```diff
@@ -104,9 +105,13 @@ const Hero = () => {
 
                     <div className="flex justify-between mt-[8px]">
                       <div className="flex gap-x-[5px]">
-                        <div className="w-[20px] h-[20px] rounded-full">
-                          {/* todos: add contributors avator here */}
-                        </div>
+                        {contributions.slice(0, 3).map(contribution => (
+                          <div className="w-[20px] h-[20px] rounded-full">
```

Comment:
Could you avoid this pattern? We have an open [issue](https://github.com/open-sauced/hot/issues/267) with this anti-pattern, a little slice of pizza 🍕